R/expect_lint.R

```diff
@@ -1,6 +1,7 @@
 #' Lint expectation
 #'
-#' This is an expectation function to test that the lints produced by `lint` satisfy a number of checks.
+#' These are expectation function to test that the lints produced by `lint` satisfy a number of checks.
+#' `expect_no_lint` is equivalent to `expect_lint` with checks set to NULL.
```

Comment:
I would phrase this in an active way (asserts that the specified linter generates no lints on the code).

Reply:
Not entirely sure how to best phrase that. Here would my my attempt. 
```
#' These are expectation functions to test specified linters on sample code in the `testthat` framework.
#' * `expect_lint` asserts that specified lints are generated.
#' * `expect_no_lint` asserts that no lints are generated. 
#' `expect_no_lint` is equivalent to `expect_lint` with checks set to NULL, and used for better readability.
```
Better? worse? Something diffrent?

Reply:
Sounds good, but I would leave out the last sentence. This makes it easier to turn passing `NULL` into a warning / an error down the road when `expect_no_lint()` is established.

Reply:
Makes sense, give me a minute.

Reply:
And in ...